frameworks/PHP/ubiquity/benchmark_config.json

```diff
@@ -26,26 +26,28 @@
         "versus": "php"
       },
       "raw": {
-            "db_url": "/Raw",
-            "query_url": "/Raw/query/",
-            "fortune_url": "/RawFortunes",
-            "update_url": "/Raw/update/",
-            "port": 8080,
-            "approach": "Realistic",
-            "classification": "Fullstack",
-            "database": "MySQL",
-            "framework": "Ubiquity",
-            "language": "PHP",
-            "flavor": "PHP7",
-            "orm": "Raw",
-            "platform": "php-fpm",
-            "webserver": "nginx",
-            "os": "Linux",
-            "database_os": "Linux",
-            "display_name": "ubiquity-raw",
-            "notes": "",
-            "versus": "php"
-          },
+        "json_url": "/Json",
+        "plaintext_url": "/Plaintext",
+        "db_url": "/Raw",
+        "query_url": "/Raw/query/",
+        "fortune_url": "/RawFortunes",
+        "update_url": "/Raw/update/",
+        "port": 8080,
+        "approach": "Realistic",
+        "classification": "Fullstack",
+        "database": "MySQL",
+        "framework": "Ubiquity",
+        "language": "PHP",
+        "flavor": "PHP7",
+        "orm": "Raw",
+        "platform": "php-fpm",
```

Comment:
Platform changed to `FPM/FastCGI` to avoid confusions in visualize mode.

Reply:
Okay, I saw